sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java

```diff
@@ -738,13 +738,13 @@ static AnalyzeHealthcareEntitiesResult getRecognizeHealthcareEntitiesResult1(Str
         // HealthcareEntityRelations
         final HealthcareEntityRelation healthcareEntityRelation1 = new HealthcareEntityRelation();
         final HealthcareEntityRelationRole role1 = new HealthcareEntityRelationRole();
-        HealthcareEntityRelationRolePropertiesHelper.setName(role1, "Qualifier");
+        HealthcareEntityRelationRolePropertiesHelper.setName(role1, "Course");
         HealthcareEntityRelationRolePropertiesHelper.setEntity(role1, healthcareEntity3);
         final HealthcareEntityRelationRole role2 = new HealthcareEntityRelationRole();
         HealthcareEntityRelationRolePropertiesHelper.setName(role2, "Condition");
         HealthcareEntityRelationRolePropertiesHelper.setEntity(role2, healthcareEntity4);
         HealthcareEntityRelationPropertiesHelper.setRelationType(healthcareEntityRelation1,
-            HealthcareEntityRelationType.QUALIFIER_OF_CONDITION);
+            HealthcareEntityRelationType.fromString("CourseOfCondition"));
```

Comment:
same question on why the need to create the type...

Reply:
same answer in https://github.com/Azure/azure-sdk-for-java/pull/22174#discussion_r652169498